src/sentry/shared_integrations/exceptions/__init__.py

```diff
@@ -169,6 +169,17 @@ class IntegrationError(Exception):
     pass
 
 
+class ExternalAPIConfigurationError(Exception):
```

Comment:
Small nitpick, maybe we make this specific to integrations in naming to match our other integration errors? Something like `IntegrationInstallationConfigurationError`?